lib/matplotlib/cm.py

```diff
@@ -594,3 +633,35 @@ def changed(self):
         """
         self.callbacks.process('changed', self)
         self.stale = True
+
+
+# The docstrings here must be generic enough to apply to all relevant methods.
+mpl._docstring.interpd.update(
+    cmap_doc="""\
+cmap : str or `~matplotlib.colors.Colormap`, default: :rc:`image.cmap`
+    The Colormap instance or registered colormap name used to map scalar data
+    to colors. This parameter is ignored for RGB(A) data.""",
+    norm_doc="""\
+norm : str or `~matplotlib.colors.Normalize`, optional
+    The normalization method used to scale scalar data to the [0, 1] range
+    before mapping to colors using *cmap*. By default, a linear scaling mapping
+    the lowest value to 0 and the highest to 1 is used. This parameter is
```

Comment:
```suggestion
    before mapping to colors using *cmap*. By default, a linear scaling is used,
    mapping the lowest value to 0 and the highest to 1. This parameter is
```
IMHO it's a bit easier to read this way.